wolox-books/src/views/SignUp.vue

```diff
@@ -1,38 +1,39 @@
 <template lang='pug'>
-  form.column.container
+  form.column.container(@submit.prevent='submit')
     img.wolox-img(src='@/assets/LogoWolox.png')
     span.books-title
       | BOOKS
-    label.label
+    label.label(for='firstName')
       | First name
-    input.input(type='text' v-model='firstName')
-    label.label
+    input.input#firstName(type='text' v-model='firstName')
+    label.label(for='lastName')
       | Last name
-    input.input(type='text' v-model='lastName')
-    label.label
+    input.input#lastName(type='text' v-model='lastName')
+    label.label(for='email')
       | Email
-    input.input(type='text' v-model='email')
-    .error(v-show='emailRequired')
+    input.input#email(type='text' v-model='$v.email.$model')
+    span.error(v-show='emailRequired')
       | Email is required
-    .error(v-show='invalidEmail')
+    span.error(v-show='invalidEmail')
       | Email is invalid
-    label.label
+    label.label(for='password')
       | Password
-    input.input(type='password' v-model='password')
-    .error(v-show='passwordRequired')
+    input.input#password(type='password' v-model='$v.password.$model')
+    span.error(v-show='passwordRequired')
       | Password is required
-    .error(v-show='invalidPassword')
-    input.input.signup-button(v-on:click='submit' type='submit' value='Sign Up')
+    span.error(v-show='invalidPassword')
+      | Password is invalid
+    button.input.signup-button(type='submit')
```

Comment:
No click event? :thinking:

Reply:
i used `form.column.container(@submit.prevent='submit')` and it's working... it's wrong?

Reply:
No, it's perfect!